src/odemis/util/cluster.py

```diff
@@ -0,0 +1,142 @@
+# -*- encoding: utf-8 -*-
+"""
+cluster.py : backport of k-means++ initialisation method
+
+The SciPy version available via default package managment of Ubuntu 18.04LT is
+v0.19.1, which is quite old and lacks new features. This file backports the
+k-means++ initialisation method for scipy.cluster.vq.kmeans2() which is
+available from SciPy v1.2.0 onwards. When support for Ubuntu 18.04LT is no
```

Comment:
"Ubuntu 18.04LT" -> "Ubuntu 18.04 LTS"  (x2)

Reply:
Thanks for catching this typo. Fixed in updated PR.